### PR TITLE
Update sign-in-loop.md

### PR DESCRIPTION
**Pull request by msbemba**
added firefox to the list

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/1386

File: Teams/teams-sign-in/sign-in-loop.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ This issue occurs if your organization uses **Trusted Sites** in Internet Explor
 
 ## Resolution
 
-Change Microsoft Edge, Internet Explorer or Google Chrome settings using administrator rights or a Group Policy object (GPO).
+Change Microsoft Edge, Internet Explorer, Mozilla Firefox or Google Chrome settings using administrator rights or a Group Policy object (GPO).
```
Comment (JohanFreelancer9) [inline Teams/teams-sign-in/sign-in-loop.md:31]:
```suggestion
Change Microsoft Edge, Internet Explorer, Mozilla Firefox, or Google Chrome settings using administrator rights or a Group Policy object (GPO).
```

File: Teams/teams-sign-in/sign-in-loop.md [hunk 2]
```diff
@@ -90,6 +90,29 @@ To change the settings by using GPO:
 1. [Download and install the Chrome administrative template](https://support.google.com/chrome/a/answer/187202/set-chrome-browser-policies-on-managed-pcs).
 2. Add the sites listed in step 3 above to the **Content settings** > **CookiesAllowedForUrls** setting.
 
+### Mozilla Firefox
+
+1. Click the **Menu** button , then select **Preferences**.
```
Comment (JohanFreelancer9) [inline Teams/teams-sign-in/sign-in-loop.md:95]:
```suggestion
1. Select the **Menu** button, then select **Preferences**.
```

File: Teams/teams-sign-in/sign-in-loop.md [hunk 3]
```diff
@@ -90,6 +90,29 @@ To change the settings by using GPO:
...
+5. On the bottom right of the window, **click Exceptions**
+6. In the **Address of website:** text box, type the websites listed below, clicking the **Allow** button after each one.
+    - `https://*.microsoft.com`
+    - `https://*.microsoftonline.com`
+    - `https://*.teams.skype.com`
+    - `https://*.teams.microsoft.com`
+    - `https://*.sfbassets.com`
+    - `https://*.skypeforbusiness.com`
+
+7. Click **Save Changes**.
```
Comment (JohanFreelancer9) [inline Teams/teams-sign-in/sign-in-loop.md:108]:
```suggestion
1. Click the **Menu** button, then select **Preferences**.
2. Select **Privacy**.
3. Check the box to the left of **Accept cookies from sites**.
4. Next to **Accept third party cookies**, select **Always** from the menu.
5. On the bottom right of the window, **select Exceptions**.
6. In the **Address of website:** text box, type the websites listed below, clicking the **Allow** button after each one.
    - `https://*.microsoft.com`
    - `https://*.microsoftonline.com`
    - `https://*.teams.skype.com`
    - `https://*.teams.microsoft.com`
    - `https://*.sfbassets.com`
    - `https://*.skypeforbusiness.com`

7. Select **Save Changes**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by simonxjx** [comment]
image

**Pull request merged**